### ISSUE TITLE
Status message on French page needs translated

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#140]
URL: https://rsams.dev.cloud.openplus.ca/fr/user/login
User Agent: Chrome v99
Language: French

Issue Description: The text needs to match to that of the language of the page. For any text that is not going to be translated a lang attribute (lang="en") will be required.

Steps to reproduce:
1. Log in to the French version of the app using the specified url above
2. From the ‘Réinitialiser votre mot de passe’ page (Reset your password), enter an invalid username and click the ‘Soumettre’ button 

Expected results: The text under ‘Message d'état’ on the ‘Trouvez des rappels, des avis et des avis de sécurité’ page should be in the language of the page.

Note: Cross-reference Test 2, step 5 in https://github.com/mmiron-HC/WST-RSAMS/issues/138. 
![image](https://user-images.githubusercontent.com/68117281/160673979-ae5a76b6-46e0-40a4-a627-ff7af6dedf3a.png)

**Comment by echouin** [comment]
@ndolter this is fixed on prod:
![image](https://user-images.githubusercontent.com/88099454/161775861-5460c40b-5886-4eb9-a892-f14d6463edc4.png)

**Comment by hydegeoff** [comment]
The issue has been fixed in PROD based on the screenshots above, but cannot be verified in DEV
